### ISSUE TITLE
Status command sometimes malforms

### ISSUE THREAD

**Issue by hdehal** [samknight/slack_applescript#22]
Hello -- thanks for this amazing Applescript!

Occasionally, it seems that Slack malforms the `/status` command as `/s. atus` which then results in an error:

<img width="792" alt="image" src="https://user-images.githubusercontent.com/21130143/126710730-041b5221-2e92-4025-a665-3c69b4c89c89.png">

* macOS `10.15.7`
* Apple Script Editor `Version 2.11 (208)`
* Spotify `1.1.64.561.g71bd09eb-a`
* slack_applescript `2.0.1 Fix`

Here is my full Applescript:
```
repeat
	if application "Spotify" is running then
		tell application "Spotify"
			set currentSong to (get artist of current track) & " – " & (get name of current track)
			tell script "Slack"
				set status currentSong with icon ":headphones:"
			end tell
		end tell
	end if
	delay 60
end repeat
```

However, if I close and restart Slack, the Apple script will work (just once). Subsequent attempts results in the malformed status.

Would appreciate your guidance here if you're able to shed some light, perhaps a mistake in my syntax -- thank you!

**Comment by samknight** [comment]
Hi I think this may have been fixed in the latest 2.0.2 version. Could you give it a try and let me know?

**Comment by hdehal** [comment]
Thank you @samknight -- the issue appears fixed!
